### PR TITLE
Added "Traditional Chinese, Hong Kong" (zh-HK) translation

### PR DESCRIPTION
**Pull request by yookoala**

### CODE REVIEW

File: rails/locales/zh-HK.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        temporarily_unavailable: 認證伺服器由於臨時負荷過重或者維護，目前未能處理請求。
+
+        #configuration error messages
+        credential_flow_not_configured: 資源擁有者密碼認證程序 (Resource Owner Password Credentials
+          flow) 失敗，原因是 Doorkeeper.configure.resource_owner_from_credentials 沒有設定。
+        resource_owner_authenticator_not_configured: 無法找到資源擁有者，原因是 Doorkeeper.configure.resource_owner_authenticator
+          沒有設定。
+
+        # Access grant errors
+        unsupported_response_type: 授權伺服器不支援這個回應類型 (response type).
```
Comment (nbulaj) [inline rails/locales/zh-HK.yml]: Last dot need to be changed to "。"

**Comment by nbulaj** [comment]
Hi @yookoala . Can you please take a look at my comment above? Also can you please place all the translations to the quotes (") ? It will prevent from some unexpected situations. After that please rebase and squash commits to a single one.

Thanks! 👍

**Comment by yookoala** [comment]
Revised the PR as required. Please check :-)

**Comment by nbulaj** [comment]
Thank you @yookoala !

**Pull request merged**